haproxy/README.md

```diff
@@ -302,3 +355,6 @@ Need help? Contact [Datadog support][20].
 [22]: https://www.datadoghq.com/blog/how-to-collect-haproxy-metrics
 [23]: https://www.datadoghq.com/blog/monitor-haproxy-with-datadog
 [24]: https://docs.datadoghq.com/integrations/faq/haproxy-multi-process/
+[25]: https://github.com/DataDog/integrations-core/blob/7.33.x/haproxy/datadog_checks/haproxy/data/conf.yaml.example
```

Comment:
```suggestion
[25]: https://github.com/DataDog/integrations-core/blob/7.34.x/haproxy/datadog_checks/haproxy/data/conf.yaml.example
```

Reply:
Nice catches. I made the requested changes